src/EnergyPlus/ChillerExhaustAbsorption.cc

```diff
@@ -1626,7 +1672,7 @@ namespace EnergyPlus::ChillerExhaustAbsorption {
                 } else {
                     ShowSevereError(state, "CalcExhaustAbsorberChillerModel: Condenser flow = 0, for Exhaust Absorber Chiller=" + this->Name);
                     ShowContinueErrorTimeStamp(state, "");
-                    ShowFatalError(state, "Program Terminates due to previous error condition.");
+                    // ShowFatalError(state, "Program Terminates due to previous error condition.");
```

Comment:
Hmmm.... OK, so if we show a severe then we need to fatal.  If we aren't going to fatal then this needs to be a warning.  But if it's a warning that could occur multiple times should we do a recurring warning message to avoid huge error files?